scripts/vcpkg-deps.rb

```diff
@@ -0,0 +1,30 @@
+# Script to generate the list of downloads for vcpkg dependencies.
```

Comment:
could this be written in a language we all understand? flathub uses python in general.

Reply:
sure, I never wrote python (more than hello world anyway) but translating such simple scripts should be easy.